vk-vtlayer-generate.py

```diff
@@ -1174,16 +1174,17 @@ def generate_intercept(self, proto, qual):
                      '    PFN_vkGetInstanceProcAddr fpGetInstanceProcAddr = chain_info->u.pLayerInfo->pfnNextGetInstanceProcAddr;\n'
                      '    PFN_vkCreateInstance fpCreateInstance = (PFN_vkCreateInstance) fpGetInstanceProcAddr(NULL, "vkCreateInstance");\n'
                      '    if (fpCreateInstance == NULL) {\n'
-                     '                return VK_ERROR_INITIALIZATION_FAILED;\n'
+                     '        (*outputStream) << "t{" << getTIDIndex() << "} vkCreateInstance FAILED TO FIND PROC ADDRESS" << endl;\n'
+                     '        return VK_ERROR_INITIALIZATION_FAILED;\n'
                      '    }\n'
                      '    // Advance the link info for the next element on the chain\n'
                      '    assert(chain_info->u.pLayerInfo);\n'
                      '    chain_info->u.pLayerInfo = chain_info->u.pLayerInfo->pNext;\n'
                      '    VkResult result = fpCreateInstance(pCreateInfo, pAllocator, pInstance);\n'
-                     '    if (result != VK_SUCCESS)\n'
-                     '        return result;\n'
-                     '    initInstanceTable(*pInstance, fpGetInstanceProcAddr);\n'
-                     '    createInstanceRegisterExtensions(pCreateInfo, *pInstance);\n'
+                     '    if (result == VK_SUCCESS) {\n'
+                     '        initInstanceTable(*pInstance, fpGetInstanceProcAddr);\n'
+                     '        createInstanceRegisterExtensions(pCreateInfo, *pInstance);\n'
+                     '    }\n'
                      '    %s%s%s\n'
```

Comment:
I think that the entire point of the change was to print out the parameters when there is a CreateInstance failure, which might allow the user to examine the parameters to determine why it failed.  

It looks like the code to print out these parameters would survive, but there's always a chance of dereferencing a non-NULL bad pointer, but at least there's a chance of getting through.

And if CreateInstance is failing, then the failure of any subsequent Vulkan calls should be expected.